eng/pipelines/templates/jobs/archetype-sdk-release.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
+  - job: 'CreateReleaseTag'
+
+    variables:
+      - template: ../variables/globals.yml
+
+    pool:
+      vmImage: '$(OSVmImage)'
+
+    steps:
+      - script: |
+          echo "TODO: Tag Azure/azure-sdk-for-ios repo!"
```

Comment:
We should use the common https://github.com/Azure/azure-sdk-for-ios/blob/master/eng/common/pipelines/templates/steps/create-tags-and-git-release.yml template but we might need to make some tweaks based to handle the ios convention.